CHANGELOG.md

```diff
@@ -75,6 +75,24 @@ If you see unexpected behaviour, make sure the `id` for the  textarea is unique
 
 This change was introduced in [pull request #2408: Prevent issues with character count when textarea `id` includes CSS syntax characters](https://github.com/alphagov/govuk-frontend/pull/2408).
 
+#### Include JavaScript for the skip link to improve screen reader announcements
+
+We've added JavaScript for the skip link component to set focus to the target element. This improves the announcements especially for Mac VoiceOver users and also for JAWS users.
+
+If you are not using Nunjucks macros, add a `data-module="govuk-skip-link"` attribute to the component HTML. For example:
+
+```html
+<div class="govuk-skip-link" data-module="govuk-skip-link">
+...
+</div>
+```
+
+If you're [importing JavaScript for individual components](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#select-and-initialise-an-individual-component), import the skip link JavaScript.
+
+Once you've made the changes, you can check that the skip link works correctly by clicking it and   inspecting the target element (commonly the `<main>` element) in the browser to ensure it has a `tabindex` attribute.
```

Comment:
```suggestion
Once you've made the changes, check that the skip link works. To make sure, click the skip link and check that the target element (usually the `<main>` element) in the browser has a `tabindex` attribute.
```

Reply:
Thanks @EoinShaughnessy, this is a nice improvement 👍  

I've gone with your wording but I've added the word 'JavaScript'. This is because strictly speaking we want to them to check that the skip link _JavaScript_ works. You might get the impression that the skip link has worked since the browser has shifted the page content but the JavaScript might have not fired - checking for the `tabindex` ensures that it did.